docs/source/cuml-accel/limitations.rst

```diff
@@ -443,6 +446,42 @@ Additionally, the following fitted attributes are currently not computed:
 
 - ``n_iter_``
 
+LinearSVC
+^^^^^^^^^
+
+``LinearSVC`` will fall back to CPU in the following cases:
+
+- If ``X`` is sparse.
+- If ``intercept_scaling`` is not ``1``.
+- If ``multi_class`` is not ``"ovr"``.
+
+The following fitted attributes are currently not computed:
+
+- ``n_iter_``
+
+Additional notes:
+
+- Sample weight functionality may not produce equivalent results to replicating data according to weights.
+- Use of sample weights may not produce exactly equivalent results when compared to replicating data according to weights.
+- Models may not be picklable; pickling or unpickling may fail.
```

Comment:
This issue concerns me - can you open an issue with a reproducer so we can investigate further?